nixos/modules/services/misc/dump1090-fa.md

```diff
@@ -0,0 +1,26 @@
+# Dump1090-fa {#module-services-dump1090-fa}
+
+[dump1090-fa](https://github.com/flightaware/dump1090) is a ADS-B, Mode S, and Mode 3A/3C demodulator and decoder that will receive and decode aircraft transponder messages received via a directly connected software defined radio, or from data provided over a network connection.
+
+## Configuration {#module-services-dump1090-fa-configuration}
+
+When enabled the module will automatically create a systemd service for starting the application making it write json files to `/run/dump1090-fa`.
+By enabling Nginx (through `services.dump1090-fa.enableNginx`) a `dump1090-fa` virtual host serving both the static web app under `/` and the dynamically generated json files from `/run/dump1090-fa/` under `/data/` is created.
```

Comment:
This option seems to be called differently now, and I'm honestly not so sure it makes sense to have it. 

Configuring "half of" a nginx vhost is a bit of a strange API. We normally have services listen on ports, or configuring the entirety of the nginx vhost (like nextcloud etc, though it produces a lot of option duplication).

I'd prefer to keep this module just writing files somewhere, remove the nginx options and include the two location aliases in the example below.

Reply:
I'd prefer only the docs, not the passthru. This is less likely to change randomly and makes the nginx config easier to understand.

Reply:
Done, I've hardcoded `/run/dump1090-fa` in the example in the manual as `locations."/data/".alias` since it's the default value, hope it's fine.